rosidl_generator_py/resource/_msg_support.entry_point.c.em

```diff
@@ -4,15 +4,33 @@
 #include <Python.h>
 #include <stdint.h>
 
-#include <rosidl_generator_c/message_type_support.h>
+@{
+static_includes = {}
+for spec, subfolder in message_specs:
+  if subfolder == 'msg':
+    static_includes[subfolder] = '#include <rosidl_generator_c/message_type_support.h>'
+  elif subfolder == 'srv':
+    static_includes[subfolder] = '#include <rosidl_generator_c/service_type_support.h>'
+for value in sorted(static_includes.values()):
+  print(value)
```

Comment:
I agree I prefer the generated code to be out of the @{}@ statement that is used to set variables/context. fixed in https://github.com/ros2/rosidl/pull/171/commits/2501e7d3dceb00a27d121f2815c46a4e987b569b